frontend/www/lib/ionic/scss/_loading.scss

```diff
@@ -0,0 +1,51 @@
+
+/**
+ * Loading
+ * --------------------------------------------------
+ */
+
+.loading-container {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+
+  z-index: $z-index-loading;
+
+  @include display-flex();
+  @include justify-content(center);
+  @include align-items(center);
+
+  @include transition(0.2s opacity linear);
+  visibility: hidden;
+  opacity: 0;
+
+  &:not(.visible) .icon,
+  &:not(.visible) .spinner{
+    display: none;
+  }
+  &.visible {
+    visibility: visible;
+  }
+  &.active {
+    opacity: 1;
+  }
+
+  .loading {
+    padding: $loading-padding;
```

Comment:
Properties should be ordered background-color, border-radius, color, font-size, padding, text-align, text-overflow